google/cloud/bigquery/routine/routine.py

```diff
@@ -16,11 +16,50 @@
 
 """Define resources for the BigQuery Routines API."""
 
+from typing import Any, Iterable, List
+
 from google.protobuf import json_format
 
 import google.cloud._helpers
 from google.cloud.bigquery import _helpers
 import google.cloud.bigquery_v2.types
+from google.cloud.bigquery_v2.types import StandardSqlField
+
+
+# Unlike related proto-plus classes from `bigquery_v2.types.standard_sql`, this class
```

Comment:
Sounds like we need to force an internal regen to get this picked up? @tswast and I need to sort this out.

Reply:
Done [here](https://github.com/googleapis/python-bigquery/pull/777), will replace the hand-written class with the generated one.